generator/CMakeLists.txt

```diff
@@ -43,7 +43,8 @@ target_link_libraries(
     google_cloud_cpp_generator
     PUBLIC google_cloud_cpp_grpc_utils google_cloud_cpp_common
            googleapis-c++::api_client_protos
-           googleapis-c++::longrunning_operations_protos absl::str_format)
+           googleapis-c++::longrunning_operations_protos absl::str_format
+           protobuf::libprotoc)
```

Comment:
Our clang-tidy build [complains](https://source.cloud.google.com/results/invocations/4de1c111-2023-4455-8064-d92df167269d/targets/cloud-cpp%2Fgithub%2Fgoogle-cloud-cpp%2Fmaster%2Fdocker%2Fclang-tidy-presubmit/log) about the formatting here. 

The relevant part of the error is here. If you can make the formatting look like this, the build should pass.

```diff
-    PUBLIC google_cloud_cpp_grpc_utils google_cloud_cpp_common
+    PUBLIC google_cloud_cpp_grpc_utils
+           google_cloud_cpp_common
            googleapis-c++::api_client_protos
-           googleapis-c++::longrunning_operations_protos absl::str_format
+           googleapis-c++::longrunning_operations_protos
+           absl::str_format
            protobuf::libprotoc)
```

Reply:
You can look at the log and fix the formatting manually, or you can run the build script locally (on a Linux workstation, with Docker enabled):

```sh
./ci/kokoro/docker/build.sh clang-tidy ./ci/check-style.sh
```

Note that it may take a while (depends on your workstation, but up to 30 minutes?) to run this the first time.

@devjgm I suspect our macOS builds do not compile with shared libraries.